app/controllers/admin/plugins_controller.rb

```diff
@@ -2,6 +2,10 @@
 
 class Admin::PluginsController < Admin::StaffController
   def index
-    render_serialized(Discourse.visible_plugins, AdminPluginSerializer, root: "plugins")
+    render_serialized(
+      Discourse.visible_plugins.sort_by { |p| p.name.downcase.gsub("discourse-", "") },
```

Comment:
Can use `String#delete_prefix`:

```suggestion
      Discourse.visible_plugins.sort_by { |p| p.name.downcase.delete_prefix("discourse-") },
```